bin/reth/src/stage/mod.rs

```diff
@@ -61,8 +59,8 @@ pub struct Command {
     /// Secret key to use for this node.
     ///
     /// This also will deterministically set the peer ID.
-    #[arg(long, value_name = "SECRET_KEY", verbatim_doc_comment)]
-    secret_key: Option<SecretKey>,
+    #[arg(long, value_name = "PATH", global = true, required = false, default_value_t)]
+    p2p_secret_key: PlatformPath<SecretKeyPath>,
```

Comment:
It looks like Geth doesn't have this option, does it?

so we're free to choose a name here, imo p2p_ or network_ prefix is fine.

Reply:
It looks like `geth` does not have this option, although I am not super familiar with that execution client.

Reply:
`geth` produces the same peer ID all the time because it uses a data directory. So the default behavior change seems to be closer to what `geth` does.

This does seem like potentially a `--datadir` like option would be the best in the end, which remaps all the default directories to the provided one.

Reply:
ah okay, in that case this behaviour should be in line with expectations

ty